lib/change-control-lambda/disable-transition.ts

```diff
@@ -1,6 +1,9 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-import * as AWS from 'aws-sdk';
-const pipeline = new AWS.CodePipeline();
+
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { CodePipeline } from '@aws-sdk/client-codepipeline';
+const pipeline = new CodePipeline();
```

Comment:
```suggestion
import { CodePipeline } from '@aws-sdk/client-codepipeline';
const pipeline = new CodePipeline();
```
formatting clean up

Reply:
Why do you think the comment line `// eslint-disable-next-line import/no-extraneous-dependencies` is there?